addons/web/static/src/webclient/actions/reports/report.scss

```diff
@@ -2,6 +2,30 @@ $o-default-report-font: 'Lato' !default;
 $o-default-report-primary-color: rgb(0, 0, 0) !default;
 $o-default-report-secondary-color: rgb(0, 0, 0) !default;
 
+// wkhtmltopdf doesn't support CSS custom properties (--name-stuff)
+// It doesn't support CSS's rgba function
+// We re-define here the .bg-[theme-colors] (and text)
+// In order to define them as a css rgb function
+// The gray scales are done in bootstrap_review.scss
+$report-extended-theme-colors: map-merge(
+  $utilities-colors,
+  (
+    "black": to-rgb($black),
+    "white": to-rgb($white),
+    "body": to-rgb($body-bg)
+  )
+);
+@each $map in ($report-extended-theme-colors, ) {
```

Comment:
This loop isn't necessary if there is only one value

Reply:
@rfr-odoo modif done